cpu/stm32f1/stmclk.c

```diff
@@ -27,7 +27,7 @@
 #include "cpu.h"
```

Comment:
can't we remove this file completely now?

Reply:
I forgot it, but we must remove it, for sure!